mapper/mapper.go

```diff
@@ -124,10 +124,16 @@ func mapEmergencyBanner(bannerData zebedee.EmergencyBanner) coreModel.EmergencyB
 
 func createPreGTMJavaScript(title string, description model.ReleaseDescription) []template.JS {
 	releaseStatus := "cancelled"
+	censusTag := ""
+
 	if description.Published {
 		releaseStatus = "published"
 	}
 
+	if description.Census2021 {
```

Comment:
So the `description` field can't be `published` and `census`? How are you going to describe published census data?

Reply:
On description attribute, both Published and Census2021 are bool flag. I our case an article can be published and censused. We are using census to display census flag on  release page.